### DISCUSSION TITLE
Creative mode

### DISCUSSION THREAD

**Discussion by Brunoasd98** [voxeltycoon/issues#820] [General] [answered]
I suggest the creative mode or something similar to test game mechanics before playing in normal mode, I wasted a lot of time trying to figure out things, such as the wooden boards, the wooden beams, for that there is no tutorial, I imagine it is Because the game is in beta, but it would be great to be able to try those things so as not to have to waste time creating new worlds, since by not knowing how to do it, you lose money and you become bankrupt

**Comment by katepolak** [comment, marked as answer]
There already is "creative mode" like feature set provided by cheats. To enable them, press CTRL+ALT+C, one they are enabled you can access the money cheats by right clicking the balance indicator in the bottom left of your screen.

**Comment by andrewpey** [comment]
We also plan to add proper Creative mode at some point.
